atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/PostgresDdlConfig.java

```diff
@@ -45,4 +45,9 @@ public final String type() {
     public <T> T accept(Visitor<T> visitor) {
         return visitor.visit(this);
     }
+
+    @Value.Default
+    public long compactIntervalMillis() {
+        return 0L;
```

Comment:
Do we want the default to be some reasonable number instead? 1 day, 3 days?

Reply:
Idea is to keep the behavior unchanged unless someone configures it, we can change the default once its tested out at one deployment and we know it works.

Reply:
Since this is a new config option, it would be nice to be able to specify it in a human readable form such as `24h`